### ISSUE TITLE
Add index page for `https://timber.github.io/`

### ISSUE THREAD

**Issue by jasalt** [timber/docs#89]
I often navigate to the docs entering to this url (from navbar history auto completion) expecting it to be the docs index page but it gives 404. 

I think it should include something or redirect to https://timber.github.io/docs/.

Currently opening https://timber.github.io/ looks like this:
![Image](https://github.com/user-attachments/assets/bbf3cbb2-fbd9-41e5-8218-0fccb0b5502a)

**Comment by gchtr** [comment]
Good idea! I just checked how to do that. And I think for this to work, we would have to create a repository named **timber.github.io** and publish an index.html there.

> To publish an organization site, you must create a repository owned by an organization that's named `<organization>.github.io`
>
> – https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites

**Comment by jasalt** [comment]
@gchtr I looked at this again, and indeed as the docs are set to live in `/docs/` path, [the redirects done there](https://github.com/timber/docs/blob/master/_data/redirects.json) cannot themselves help with the issue as they work only relative to `/docs/`.

So a new repository for Github Pages seems to be the simplest option having an `index.html` that redirects to the `/docs/` or directly to `/docs/v2/`. The current auto-generated redirect html in `/docs/` should be sufficient:

```html
<!doctype html><html lang="en"><head><meta http-equiv="refresh" content="0; url=/docs/v2/"><title>Redirect</title></head><body></body></html>
```

**Comment by gchtr** [comment]
I just set this up and it seems to be working: https://timber.github.io/. 🚀
